README.md

```diff
@@ -1,38 +1,139 @@
+
 # NiftyOne
 
 ![NiftyOne Mosaic](.github/static/niftyone_mosaic_view.png)
 
 NiftyOne is a platform for bulk visualization of large-scale neuroimaging datasets. It is built with [FiftyOne](https://docs.voxel51.com/).
 
-## Installation
+- [NiftyOne](#niftyone)
+  * [Installation](#installation)
+  * [Testing](#testing)
+  * [Usage](#usage)
+  * [Running NiftyOne on a remote cluster](#running-niftyone-on-a-remote-cluster)
+  * [FAQ](#faq)
+
 
+## Installation
+**1. Install [conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html)**
 For stability, NiftyOne should be installed in it's own environment. For example,
 
 ```bash
 conda create -y -n niftyone python=3.10
 conda activate niftyone
 
 pip install -U pip
-pip install -r https://raw.githubusercontent.com/cmi-dair/niftyone/main/requirements.txt
 pip install git+https://github.com/cmi-dair/niftyone.git
 ```
-
+## Testing
+1. Install [datalad](https://www.datalad.org) to download the data files
+2. You can use whatever BIDS dataset (such as [ds000102](https://openneuro.org/datasets/ds000102/versions/00001)  from OpenNeuro) as an example to test 
+```bash
+niftyone bids_dir output_dir participant --workers 8
+```
 ## Usage
+### 1. Test launching the app
+```bash
+niftyone app launch
+```
 
-### 1. Generate figures for each participant
+### 2. Generate figures for each participant
 
 ```bash
 niftyone bids_dir output_dir participant --workers 8
 ```
 
-### 2. Collect participant figures into a FiftyOne dataset
+### 3. Collect participant figures into a FiftyOne dataset
 
 ```bash
 niftyone bids_dir output_dir group
 ```
 
-### 3. Launch FiftyOne app
+### 4. Launch FiftyOne app
 
 ```bash
 niftyone bids_dir output_dir launch
 ```
+## Running NiftyOne on a remote cluster
```

Comment:
This gets a little in the weeds. I might consolidate into the above steps as **5. Running NiftyOne remotely**. Then just explain how you can launch niftyone remotely as usual and forward the port.  

You can assume a simple remote server, i.e. not a cluster requiring compute node reservation, multi-hop forward, etc. Pointers for how to adapt to the cluster/multi-hop case can come at the end.